cmd/daemon/kubernetes/pods.go

```diff
@@ -0,0 +1,209 @@
+package kubernetes
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"math"
+	"strings"
+
+	httpinternal "github.com/lunarway/release-manager/internal/http"
```

Comment:
Again, no need for the import alias here

```suggestion
	"github.com/lunarway/release-manager/internal/http"
```